scripts/install/nzbget.sh

```diff
@@ -45,31 +45,19 @@ function _install() {
     cd /tmp
     for u in "${users[@]}"; do
         echo_progress_start "Installing nzbget for $u"
-        sh nzbget-latest-bin-linux.run --destdir /home/$u/nzbget >> $log 2>&1
```

Comment:
This cannot be changed without refactoring the way multiple users are handled by this script